src/core/filters/filterregistry.js

```diff
@@ -57,18 +56,65 @@ export default class FilterRegistry {
   }
 
   /**
-   * Gets the filter string to send in a search query.
-   * TODO: move payload method logic into core.js, since it is only used there.
-   * @returns {string}
+   * Gets the static filters as a {@link CombinedFilter} to send to the answers-core
    */
   getStaticFilterPayload () {
-    return JSON.stringify(this._getStaticFilterPayload());
+    const filterNodes = this.getStaticFilterNodes();
+
+    const transformedFilterNodes = filterNodes.map(filterNode => {
+      return this._transformFilter(filterNode);
+    }).filter(filter => filter);
+
+    if (transformedFilterNodes.length > 1) {
+      return {
+        combinator: '$and',
+        filters: transformedFilterNodes
+      };
+    }
+    return transformedFilterNodes[0];
   }
 
-  _getStaticFilterPayload () {
-    const filterNodes = this.getStaticFilterNodes();
-    const totalNode = FilterNodeFactory.and(...filterNodes);
-    return totalNode.getFilter();
+  _transformFilter (filterNode) {
+    if (filterNode.children) {
+      return this._recurseCombinedFilterNodes(filterNode.children, filterNode.combinator);
+    }
+
+    // All static filters come through this code path, even if the filters have not been
+    // interacted with. We should ignore filters that effectively don't filter anything.
+    if (!filterNode.filter || Object.entries(filterNode.filter).length === 0) {
```

Comment:
Instead of having this check here, should `getStaticFilterNodes` simply not return such `filterNode`s?

Reply:
Looking into this! `getStaticFilterNodes` returns what's in global storage, which appears to be one node per static filter on the page. If static filter nodes were only simple filters, I'd say definitely yes, but since combined filters can be nested, it makes it a little trickier to know if they contain filtering data

Reply:
Update here so far - I'm a little hesitant to change the behavior of `getStaticFilterNodes` since it's used in the ResultHeader and AppliedQueryFilters. Going to try to make the change and test things out to see what happens!

Reply:
@oshi97, do you foresee any problems updating `getStaticFilterNodes` in this way?

Reply:
Instead of updating `getStaticFilterNodes`, could we put this filtering logic in `getStaticFilterPayload`? 
```
const transformedFilterNodes = filterNodes
    .filter(filterNode => ...)
    .map(this.transformFilter);
```
If we need this filtering in `_recurseCombinedFilterNodes`, maybe we could make a shared validation function. It just feels a bit odd to put the validation in `_transformFilter`, I think it should be up to the caller to pass it a proper `filterNode`. I think we should avoid returning `undefined` from a method when we can. When other functions begin to interpret an `undefined` output as meaning something, that can get a bit brittle.

Reply:
Oh I like that! Yes! I'll try that out now

Reply:
I think CombinedFilterNodes will never have any empty filter nodes because in FilterNodeFactory any SimpleFilterNodes that are empty are filtered out. I... think this is something that's safe to assume though we might want some kind of refactor later since you ARE able to break this rule if you don't use FilterNodeFactory and call the constructors directly.

I can't remember exactly why we let you store empty filter nodes, but I think having a validation step makes sense!

Reply:
you could probably just do `filterNodes.filter(fn => fn.getFilter().getFilterKey());` (ripped straight from FilterNodeFactory)